lib/round.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/turn'
+class Round
+  attr_reader :deck, :turns, :new_turn, :number_correct, :number_of_turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @number_of_turns = 0
+  end
```

Comment:
you only need attr_readers for the attributes that you want to be able to read outside of this class. So, for that reason, you only need to have attr readers for deck and turns.